calico_containers/pycalico/util.py

```diff
@@ -0,0 +1,12 @@
+def generate_cali_interface_name(prefix, ep_id):
```

Comment:
No, I created this commonized method by pulling out the logic from the one place that it was inlined.